README.md

```diff
@@ -7,28 +7,35 @@
 
 Please refer to the sections below for more detailed information. The instructions assume the work is performed from a command line.
 
+### Getting Started
+
 1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. *WordPress for iOS* requires Xcode 11.2.1 or newer.
-1. From a command line, `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
-1. `cd WordPress-iOS` to enter the working directory.
-1. `rake dependencies` to install all dependencies required to run the project (this may take some time to complete).
-1. `rake xcode` to open the project in Xcode. 
-1. Compile and run the app on a device or an simulator.
+1. From a command line, run `git clone git@github.com:wordpress-mobile/WordPress-iOS.git` in the folder of your preference.
+1. Now, run `cd WordPress-iOS` to enter the working directory.
 
-In order to login to WordPress.com using the app:
+### Create WordPress.com API Credentials
```

Comment:
Using the same heading level here as above, `###`, made me think this was a different section, while it's a subsection of the setup instructions
```suggestion
#### Create WordPress.com API Credentials
```